extensions/github1s/src/adapters/ossinsight/templates.ts

```diff
@@ -0,0 +1,249 @@
+/**
+ * @file Trending page
+ * @author netcon
+ */
+
+import { trimEnd } from '@/helpers/util';
+import { RankingLanguages, RankingPeriod } from './constants';
+import {
+	getCollections,
+	getCollectionIdByName,
+	getCollectionIssuesLast28DaysRank,
+	getCollectionPullRequestsLast28DaysRank,
+	getCollectionStarsLast28DaysRank,
+	getRecentHotCollections,
+	getTrendingRepos,
+	RepoItem,
+} from './interfaces';
+
+const buildRepoLink = (repo: string) => `https://github1s.com/${repo}`;
+
+const dataSourceMarkdown = ` (Data source: [OSS Insight](https://ossinsight.io/))`;
+
+const createRepoItemMarkdown = (repo: RepoItem, period: RankingPeriod) => {
+	const contributorAvatars = (repo.contributor_logins?.split(',') || []).map((username) => {
+		// too many avatars may trigger github rate limits
+		// return `[<img class="avatar" src="https://github.com/${username}.png" alt="${username}" width="18" style="border-radius: 50%; vertical-align: middle;">](https://github.com/${username})`;
+		return `[${username}](https://github.com/${username})`;
+	});
+	const repoBadges = [
+		`![Language](https://img.shields.io/github/languages/top/${repo.repo_name})`,
+		`![Watch](https://img.shields.io/github/watchers/${repo.repo_name}?label=Watch)`,
+		`![Fork](https://img.shields.io/github/forks/${repo.repo_name}?label=Fork)`,
+		`![Star](https://img.shields.io/github/stars/${repo.repo_name}?label=Star)`,
+		`![LastCommit](https://img.shields.io/github/last-commit/${repo.repo_name})`,
+	];
+
+	const increaseStarsText = (repo.stars || 0) >= 0 ? `+${repo.stars || 0}` : `-${repo.stars}`;
+	const increaseForksText = (repo.forks || 0) >= 0 ? `+${repo.forks || 0}` : `-${repo.forks}`;
+	const contributorsMardown = contributorAvatars.length
+		? ' &nbsp;&nbsp; Built by &nbsp;' + contributorAvatars.join('&nbsp;&nbsp;')
+		: '';
+	const collectionMarkdown = repo.collection_names ? ` &nbsp;&nbsp; <i>${repo.collection_names}</i>` : '';
+	const fireThreshold = period === RankingPeriod.ThisWeek ? 5000 : period === RankingPeriod.ThisMonth ? 12000 : 1000;
+
+	return `
+## ${(repo.total_score || 0) >= fireThreshold ? '🔥' : '🚀'} [${repo.repo_name}](${buildRepoLink(repo.repo_name)})
+
+${repo.description || ''}
+
+⭐️ ${increaseStarsText} &nbsp;&nbsp; 🔗 ${increaseForksText} ${contributorsMardown}${collectionMarkdown}
```

Comment:
Instead of 🔗, can we `import 'vscode'`  & use`$(git-branch)`. Then, it would look like Branch/fork kind of icon.

Reply:
I don't think, that would pe possible. We can skip that.